src/main/webapp/Dockerfiles

```diff
@@ -1,2 +0,0 @@
-FROM gcr.io/google_appengine/jetty9-compat:githubheadasync
```

Comment:
I don't understand the rationale behind the changes in this file. Could you provide some context?

Reply:
It was used to customize the environment. But it's not needed anymore.